service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -131,32 +135,57 @@ public AboutToStartOrSubmitCallbackResponse handlePopulateSelectorMidEvent(
 
     @PostMapping("/generate-document/mid-event")
     public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody CallbackRequest callbackRequest) {
-        CaseDetails caseDetails = callbackRequest.getCaseDetails();
-        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+        // This mid event is called after:
```

Comment:
Probably should java doc this one too

Reply:
Don't think it really fits as java doc but can happily make it a multiline comment